File: lib/models/court.dart
class Court {
  final int? courtID;
  final String courtName;
  final String? courtType;
  final String? address;
  final String? phone;

  Court({
    this.courtID,
    required this.courtName,
    this.courtType,
    this.address,
    this.phone,
  });

  factory Court.fromMap(Map<String, dynamic> map) {
    return Court(
      courtID: map['courtID'] ?? map['CourtID'],
      courtName: map['courtName'] ?? map['CourtName'] ?? '',
      courtType: map['courtType'] ?? map['CourtType'],
      address: map['address'] ?? map['Address'],
      phone: map['phone'] ?? map['Phone'],
    );
  }

  Map<String, dynamic> toMap() {
    return {
      'CourtID': courtID,
      'CourtName': courtName,
      'CourtType': courtType,
      'Address': address,
      'Phone': phone,
    };
  }
}

File: JudiciaryProject/JudiciaryProject/Models/Court.cs
namespace JudiciaryApi.Models
{
    public class Court
    {
        public int CourtID { get; set; }
        public string CourtName { get; set; } = string.Empty;
        public string? CourtType { get; set; }
        public string? Address { get; set; }
        public string? Phone { get; set; }
    }
}
